src/module/Ui/templates/layout/en/serlo-home.phtml

```diff
@@ -89,75 +89,122 @@ echo $this->doctype();
                 <?php echo $this->partial('layout/partials/notifications'); ?>
 
                 <div class="flasher"></div>
+                <div class="home-header">
+                    <?php echo $this->brand()->getLogo(); ?>
+                    <?php echo $this->brand()->getBrand(); ?>
+                    <div class="header-text">The free Learning platform</div>
+                </div>
 
-                <div class="home-serlo-row home-row">
-                    <div class="home-column">
-                        <div class="logo-wrapper">
-                            <span class="serlo-logo">V</span>
-                            <div class="serlo-brand">Serlo</div>
-                        </div>
-
+                <div class="home-serlo-row">
+                    <div class="home-row">
                         <p>
-                            It is our vision to provide high quality education worldwide -
-                            completely free of charge and without ads. Like on Wikipedia,
-                            every user can create new content or improve upon existing materials.
-                            We already provide thousands of articles and solved exercises in German -
-                            now it’s time to go international.
-                        </p>
+                            It is our vision to enable personalized learning and provide high
+                            quality educational resources<br>worldwide &ndash; completely free of charge.
```

Comment:
Hm thing is - without the manual line break, the text looks awkward for most screen sizes. I would prefer to keep the n-dash